pkg/goDB/conditions/tokenize.go

```diff
@@ -45,12 +45,11 @@ import (
 // NOTE:  the current implementation of GPDPIProtocols.go has to make sure that the map keys
```

Comment:
We probably can get rid of this comment (totally unrelated, just caught my eye) as there is no DPI stuff anymore...